Chat: Add Google Dialogflow Integration topic (#8231)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chat/15 Integrate with AI Service/10 Google Dialogflow/10 Server Configuration.md

> Dialogflow requires a server layer. You cannot use this library on the client exclusively since Google's CORS policy blocks necessary requests. 

Reviewer: ```suggestion
Dialogflow requires a server layer. You cannot use this library on the client only because Google’s CORS policy blocks required requests.
```

> Google recommends to implement a server layer when you use Dialogflow to ensure the security and reliability of the application. You cannot use this library on the client exclusively since Google's CORS policy blocks the necessary requests. 

Reviewer 2: At first, a server layer looks optional. You say "recommends". You emphasize advantages ("security" and "reliability"). When I read all this I still think that I can get away without a server layer. My app wouldn't be as secure and reliable as possible, but it will work. But then the last sentence says "Goolge blocks necessary requests". So it's not a recommendation then? It's a requirement, as I understand. Otherwise Diagflow simply doesn't function as it should.

So I would switch from recommendation to a requirement:

```suggestion
Dialogflow requires a server layer. You cannot use this library on the client exclusively since Google's CORS policy blocks necessary requests. 
```

To create a server layer, perform the following steps:

1. Create a `server` folder. It should include your `key.json`, `server.js`, and `dialogflow.js` files (see the configuration in the [GitHub example repository](https://github.com/DevExpress-Examples/devextreme-chat-google-dialogflow/tree/25.1.2%2B/server)).

2. [Create `package.json`](https://docs.npmjs.com/creating-a-package-json-file) and add the following script:

        <!-- tab: package.json -->
        {
            // ...
            "scripts": {
                "start:server": "node server.js"
            }
        }

> 3. Install the necessary dependencies:

Reviewer 2: ```suggestion
3. Install the following dependencies:
```
